docs/services/hamamatsu_camera.rst

```diff
@@ -37,27 +37,22 @@ Configuration
       sensor_width: 4096
       sensor_height: 2304
       exposure_time: 8294.4
-      gain: 0
 
 Properties
 ----------
 ``exposure_time``: Exposure time of the camera in microseconds.
 
-``gain``: Gain of the camera.
+``width``: The width of the camera frames (int).
```

Comment:
We didn't have types here before, so why add them now? If anything, units are what should be mentioned in the docs instead. It's all pixels here so I think you can save yourself the work.